src/Query/Filters/Offer.ts

```diff
@@ -47,3 +68,38 @@ export const collectionOfferQueryFilter: TQueryFilter<
   OfferFilters,
   OffersSortInput
 > = anyOfferQueryFilter("collection_offer")
+
+export const sortOffersAndCollectionOffers = (
+  offersA: AnyOffer[],
+  offersB: AnyOffer[],
+  sort: OffersSortInput
+) => {
+  const sortProperty = Object.keys(sort)[0]
+  const sortDirection = sort[sortProperty]
+
+  // workaround for the floorDifference computed column
+  if (sortProperty === "floorDifference") {
```

Comment:
need to have a manual workaround for floorDifference as I can't get typeorm to return it from the query 😕

Reply:
let's validate for the time being and extract it as a github issue to tackle on eventually